Doc/c-api/exceptions.rst

```diff
@@ -402,51 +402,44 @@ Querying the error indicator
 
 .. c:function:: PyObject *PyErr_GetRaisedException(void)
 
-   Returns the exception currently being raised, clearing the exception at
-   the same time. Do not confuse this with the exception currently being
-   handled which can be accessed with  :c:func:`PyErr_GetHandledException`.
+   Return the exception currently being raised, clearing the error indicator at
+   the same time.
 
-   .. note::
+   This function is used by code that needs to catch exceptions,
+   or code that needs to save and restore the error indicator temporarily.
 
-      This function is normally only used by code that needs to catch exceptions or
-      by code that needs to save and restore the error indicator temporarily, e.g.::
+   For example::
 
-         {
-            PyObject *exc = PyErr_GetRaisedException();
+      {
+         PyObject *exc = PyErr_GetRaisedException();
 
-            /* ... code that might produce other errors ... */
+         /* ... code that might produce other errors ... */
 
-            PyErr_SetRaisedException(exc);
-         }
+         PyErr_SetRaisedException(exc);
+      }
 
-   .. versionadded:: 3.12
+   .. seealso::
 
+      * :c:func:`PyErr_SetRaisedException`,
+        to set the current raised exception.
```

Comment:
As mentioned previously, I would suggest eliminating this one as it is listed on the very next line—it would be rather noisy to list the many directly adjacent related functions like this.

```suggestion
   .. seealso:: :c:func:`PyErr_GetHandledException`,
                to fetch the exception currently being handled.
```

Or, if it really does need both, as [officially documented](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-seealso) and done elsewhere, a See-Also should have its contents formatted as a definition list:

```suggestion
   .. seealso::

      :c:func:`PyErr_SetRaisedException`
         To set the current raised exception.

      :c:func:`PyErr_GetHandledException`
         To fetch the exception currently being handled.
```

Reply:
I'm not sure if we need the cross-linking between the set/get APIs. IMO, it leads to too much clutter.